hack/jenkins/test-dockerized.sh

```diff
@@ -55,6 +55,8 @@ make generated_files
 go install ./cmd/...
 ./hack/install-etcd.sh
 
+# bazel didn't like BUILD files in the staging repos, so we need to run unit tests
+make test WHAT="./vendor/k8s.io/apimachinery/... ./vendor/k8s.io/apiserver/... ./vendor/k8s.io/client-go/..."
```

Comment:
shouldn't we instead set up CI on these repos individually, rather than running their unit tests here?